docs/Getting-Started/Quickstart.md

```diff
@@ -0,0 +1,126 @@
+# Getting Started
+
+## Prequisites
+
+[Orion](../Installation/Install-Binaries.md)
+
+## Starting Orion and Sending a Payload
+
+To start Orion and send a payload: 
+1. Generate Keys 
+1. Create Password File
+1. Create a Configuration File 
+1. Start Orion 
+1. Confirm Orion is Running 
+1. Send a Payload
+1. Receive a Payload 
+
+### 1. Generate Keys
+
+To generate a public/private keypair for the Orion node:
+
+``` bash
+orion -g orion
+```
+
+At the prompt, enter a password to encrypt the key pair. 
+
+The public/private key pair is generated and the keys saved in the `orion.pub` and `orion.key` files.
+
+### 2. Create Password File
```

Comment:
I think we should have this step before the previous step (generating keys). I can see someone following the guide step by step without realising that the password needs to be something they'll need in the future.

Reply:
Done